error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/AmbiguousJsonCreator.java

```diff
@@ -36,6 +36,9 @@ public final class AmbiguousJsonCreator extends BugChecker implements Annotation
   private static final Matcher<AnnotationTree> IS_JSON_CREATOR_ANNOTATION =
       isType("com.fasterxml.jackson.annotation.JsonCreator");
 
+  /** Instantiates the default {@link AmbiguousJsonCreator}. */
```

Comment:
"The default" hints at there being a non-default instance, while in fact there can be zero or more equivalent instances. How about:  
```suggestion
  /** Instantiates a new {@link AmbiguousJsonCreator} instance. */
```

(The three existing cases should be named "a default", as a contrast with "a customized".)

Reply:
Aha, I see. I used the same sentence that we already used in a few places 😬.